packages/components/src/DataList/components/DataListOverflowFade/DataListOverflowFade.css

```diff
@@ -1,6 +1,5 @@
 .fadeContainer {
-  --overflow-fade--offset: var(--space-smaller);
-  --overflow-fade--negative-offset: calc(var(--overflow-fade--offset) * -1);
```

Comment:
These values are now the same and can be applied accordingly with one variable

I considered removing the margin and padding declarations, but since the positions of the gradients are coupled to those values, felt worth keeping them in sync